tensorflow/lite/micro/tools/make/targets/ceva_makefile.inc

```diff
@@ -0,0 +1,110 @@
+
+ifeq ($(TARGET), ceva)
+
+TARGET_ARCH :=
+
+ifeq ($(TARGET_ARCH), )
+  $(error TARGET_ARCH must be specified on the command line)
+
+endif# Create a cflag based on the specified TARGET_ARCH. For example:
+#   TARGET_ARCH=hifimini --> -DHIFIMINI
+#   TARGET_ARCH=fusion_f1 --> -DFUSION_F1
+TARGET_ARCH_DEFINES := -D$(shell echo $(TARGET_ARCH) | tr [a-z] [A-Z])
+
+ifeq ($(TARGET_ARCH), CEVA_BX1)
+ #TARGET_ARCH := 
+CXXFLAGS := -std=c++11 -DTF_LITE_STATIC_MEMORY -I./tensorflow/lite/micro/kernels/ceva
+CCFLAGS  := -std=c11   -DTF_LITE_STATIC_MEMORY -I./tensorflow/lite/micro/kernels/ceva
```

Comment:
let's avoid the additional include path if we can. If not (for the existing code), then that's ok.